_shows/21_22/home:_musicality.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Home: Musicality
+playwright: Amber India Frost and Ethan Lee
+period: Spring
+season: StuFF
+season_sort: 380
+date_start: 2022-06-14
+date_end: 2022-06-14
+venue: New Theatre
+
+crew:
```

Comment:
Is that it? Non-NNT crew can be included and if they defo aren't going to be members, set `person: false`

Reply:
i cant find any crew  other than the playwrights but did add cast